README.md

```diff
@@ -42,3 +42,312 @@ Run linters.
 cargo make clippy
 ```
 
+## Usage
+
+In this section is presented the integration of this library into your application, enabling you to kickstart 
```

Comment:
```suggestion
In this section we present how to integrate this library into your application, enabling you to kickstart 
```

Reply:
I like much more to leave it impersonal. What the others think?